### DISCUSSION TITLE
Ryzen AI NPU simulator in Riallto

### DISCUSSION THREAD

**Discussion by hecmay** [AMDResearch/Riallto#23] [Q&A] [answered]
Hi,

I am interested in trying out Riallto. However, I do not have a Ryzen AI NPU hardware. Is there a simulator for NPU that I can use?

Thanks

**Comment by STFleming** [comment, marked as answer]
Hi @hecmay,

I'm afraid that there is no publicly available simulator that can be used with Riallto currently.

All the best,
Shane

**Comment by mariodruiz** [comment]
Hi @hecmay,

If you are attending FCCM24 next week, you will have a chance to access Riallto in the AMD tutorial.

**Reply by hecmay** [reply]
Thanks @mariodruiz . That's great! We will come over and attend the tutorial.
